modules/ossm-cr-example.adoc

```diff
@@ -25,6 +27,7 @@ The 3scale Istio Adapter is deployed and configured in the custom resource file.
   kind: ServiceMeshControlPlane
   metadata:
     name: basic-install
```

Comment:
Should we update this SMCP definition to be the full-install example?  Would this be more beneficial to customers?

Reply:
@travisrogers05 Feel free to provide a full-install example and I will add it to the docs.

Reply:
If we're going to link to one, I'd suggest this one which is public:

https://github.com/Maistra/istio-operator/blob/maistra-1.0/deploy/examples/maistra_v1_servicemeshcontrolplane_cr_full.yaml

Reply:
I've added the full example that @brian-avery shared. If this needs to be modified, we can do that in a post-GA update.